front/src/modules/pathfinding/hook/usePathfinding.ts

```diff
@@ -285,6 +285,7 @@ export const usePathfindingV2 = (
               );
 
               setPathProperties?.({
+                manchetteOperationalPoints: operational_points,
```

Comment:
You can remove this line since usePathfinding is not use in the scenario results not in the stdcm results
```suggestion
```